docs/operations/cleaning_up_the_swarm.rst

```diff
@@ -0,0 +1,133 @@
+.. _operations-cleaning-up-the-swarm:
+
+##################
+Clean Up The Swarm
+##################
+
+Docker swarm does not come with any "garbage collection" for dangling 
+volumes or unused containers that have been created during updates or after 
+restarts.
+
+**To Do: Add Cronjob to run** ``docker prune``
```

Comment:
Minor: I think we can remove this TODO as the ansible playbook we added creates this cron job